src/main/java/eu/dissco/core/digitalspecimenprocessor/property/RabbitMqProperties.java

```diff
@@ -106,10 +106,10 @@ public static class DigitalMediaRelationshipTombstone {
   public static class CreateUpdateTombstone {
 
     @NotBlank
-    private String exchangeName = "create-update-tombstone-exchange";
+    private String exchangeName = "provenance-exchange";
 
     @NotNull
-    private String routingKeyName = "create-update-tombstone";
+    private String routingKeyName = "provenance";
```

Comment:
In the deployment pr, you have the routing key for this queue "provenance.#"
Does rabbitmq automatically interpret that as "provenance" in this context?